### PR TITLE
reduces clutter of mongo logs

### PR DESCRIPTION
**Pull request by JBAhire**
Mongo logs bunch of unnecessary thing, this PR:
- sets `--quiet` flag for mongo to reduce mongo logs.

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -195,10 +195,11 @@ services:
     image: mvertes/alpine-mongo
     container_name: mongo
     healthcheck:
-      test: ["CMD", "mongo", "--eval", "db.adminCommand('ping')"]
+      test: ["CMD", "nc", "-z", "localhost", "27017"]
```
Comment (adriancole) [inline docker/docker-compose.yml:198]:
```suggestion
      # reduces log output vs `mongo --eval db.adminCommand('ping')
      test: ["CMD", "nc", "-z", "localhost", "27017"]
```

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -195,10 +195,11 @@ services:
...
       interval: 5s
       timeout: 2s
       retries: 3
+    command: sh -c  "mongod --quiet | egrep -v 'connections now open'"
```
Comment (adriancole) [inline docker/docker-compose.yml:202]:
```suggestion
    # overrides normal entrypoint to quiet logs, notably request logging from HEALTHCHECK
    command: sh -c  "mongod --quiet | egrep -v 'connections now open'"
```

**Review by adriancole — DISMISSED** [review]
LGTM

Update the description with resultant `docker logs mongo` after clicking the UI?

**Comment by adriancole** [comment]
everything ok now?

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -185,10 +185,13 @@ services:
     image: mvertes/alpine-mongo
     container_name: mongo
     healthcheck:
+      # reduces log output vs `mongo --eval db.adminCommand('ping')
```
Comment (kotharironak) [inline docker/docker-compose.yml:188]: Do we need this?

**Review by kotharironak — APPROVED** [review]

**Comment by adriancole** [comment]
thanks for the help @JBAhire and @laxmanchekka I made this comment as mongo is forever disappointing https://github.com/hypertrace/entity-service/issues/18#issuecomment-682253330

**Pull request merged**